instance.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_ami" "amzn-ami" {
+    most_recent = true
+    filter{
+      name = "name"
+      values = ["amzn2-ami-hvm-*-gp2"]
+    }
+    
+    owners = ["137112412989"] #AWS
+}
+
+resource "aws_security_group" "allow_ssh" {
+    name = "allow_ssh"
+    description = "Allow SSH inbound traffic"
+    vpc_id = aws_vpc.core-infra.id
+
+    ingress {
+      description = "TLS from VPC"
+      from_port = 22
+      to_port = 22
+      protocol = "ssh"
+    }
+
+    egress {
```

Comment:
Please create a seperate security group call `egress-all` and put the `egress` block in there instead of bundling both `ingress` and `egress` in one security group.